### PR TITLE
Issue 865

### PR DESCRIPTION
**Pull request by jmillan**

### CODE REVIEW

File: worker/src/RTC/SimpleConsumer.cpp [hunk 1]
```diff
@@ -269,10 +270,9 @@ namespace RTC
 		bool marker;
 
 		// Process the payload if needed. Drop packet if necessary.
-		if (!packet->ProcessPayload(this->encodingContext.get(), marker))
+		if (this->encodingContext && !packet->ProcessPayload(this->encodingContext.get(), marker))
 		{
-			MS_DEBUG_TAG(
-			  rtp,
+			MS_DEBUG_DEV(
```
Comment (jmillan) [inline worker/src/RTC/SimpleConsumer.cpp:275]: The previous log was too noisy.

**Review by ibc — APPROVED** [review]

**Comment by ibc** [comment]
Still a draft PR?

**Comment by nazar-pc** [comment]
Can you bump worker's Rust version to 0.4.1 so I can release Rust update as well?

**Comment by ibc** [comment]
Bump what where? I'm gonna do a release right now so if you specify that I'll change it

**Comment by ibc** [comment]
Do you mean this?

```diff
--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -45,7 +45,7 @@ version = "0.7.2"
 
 [dependencies.mediasoup-sys]
 path = "../worker"
-version = "0.4.0"
+version = "0.4.1"
```

**Comment by nazar-pc** [comment]
Yes

**Comment by nazar-pc** [comment]
Actually in `worker/Cargo.toml`, that file you can leave as is

**Comment by ibc** [comment]
done in 3.10.3

**Comment by nazar-pc** [comment]
Published mediasoup-sys 0.4.1 too

**Pull request merged**